extensions/ql-vscode/src/databases/db-manager.ts

```diff
@@ -75,8 +75,18 @@ export class DbManager {
     await this.dbConfigStore.updateExpandedState(newExpandedItems);
   }
 
-  public async addNewRemoteRepo(nwo: string): Promise<void> {
-    await this.dbConfigStore.addRemoteRepo(nwo);
+  public async addNewRemoteRepo(
+    nwo: string,
+    parentList?: string,
+  ): Promise<void> {
+    if (nwo === "") {
```

Comment:
I think these checks should happen in the `DbConfigStore` if they aren't already, and the `DbManager` should just pass through to that. That's what we did elsewhere.

Reply:
Hm, this must've sneaked in during a rebase. Great..